admin/pageaudit/report/models.py

```diff
@@ -93,26 +112,95 @@ class Url(models.Model):
 
     url = models.URLField(unique=True)
     lighthouse_run = models.ForeignKey('LighthouseRun',
-                                        related_name='url_lighthouse_run',
-                                        on_delete=models.SET_NULL,
-                                        null=True, blank=True)
+                                       related_name='url_lighthouse_run',
+                                       on_delete=models.SET_NULL,
+                                       null=True,
+                                       blank=True)
     url_kpi_average = models.ForeignKey('UrlKpiAverage',
                                         related_name='url_url_kpi_average',
                                         on_delete=models.SET_NULL,
-                                        null=True, blank=True)
+                                        null=True,
+                                        blank=True)
     inactive = models.BooleanField(default=False)
     sequence = models.PositiveIntegerField(default=0)
-    
+    owner = models.ForeignKey(UrlOwner,
+                              related_name='url_owner_url',
+                              on_delete=models.SET_NULL,
+                              null=True,
+                              blank=True)
+
+    # based on https://developer.mozilla.org/en-US/docs/Web/API/Location
```

Comment:
This is a `Location` representation of the Url string - just like in the DOM. We can search by any part of it.